common/cuda_hip/matrix/dense_kernels.hpp.inc

```diff
@@ -392,6 +392,39 @@ __global__ __launch_bounds__(default_block_size) void fill_in_sellp(
 }
 
 
+template <typename ValueType, typename OutputType, typename IndexType>
+__global__ __launch_bounds__(default_block_size) void row_scatter(
```

Comment:
Do we need this kernel for `index_set`? I don't see it used anywhere else in practice